src/intervals/conversion_promotion.jl

```diff
@@ -19,3 +11,103 @@ promote_rule{T<:Real, S<:Real}(::Type{Interval{T}}, ::Type{S}) =
 
 promote_rule{T<:Real}(::Type{BigFloat}, ::Type{Interval{T}}) =
     Interval{promote_type(T, BigFloat)}
+
+
+## Conversion rules
+
+# Just specifying Interval defaults to Float64 interval
+# Remove this default so that it is necessary to explicitly specify?
+convert(::Type{Interval}, x::Real) = convert(Interval{Float64}, x)
```

Comment:
I think a reasonable default is: 

- for `x::Rational{T}` create an `Interval{Rational{T}}`;
- for `x::BigFloat`, create `Interval{BigFloat}`; 
- for the rest of `AbstractFloat`s create `Interval{Float64}`.

If someone wants to do something else more specific, it should be explicitly given.